sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/viewstate/messageholder/LayoutState.kt

```diff
@@ -82,6 +101,20 @@ internal sealed class LayoutState private constructor() {
                     get() = amount.unit
             }
 
+            data class Invoice(
+                val showSent: Boolean,
+                val amount: Sat,
+                val text: String,
+                val isExpired: Boolean,
+                val isPaid: Boolean,
```

Comment:
Does not describe the Layout's state. `showXXXConstraintLayout` is more descriptive and moves all logic from where view is set, to the instantiation of this layout state.

Reply:
@05nelsonm requested changes are done + some fixes for small bugs I found while testing